docs/embedded/concepts/spe-copilot.md

```diff
@@ -0,0 +1,262 @@
+
+### Application Scoping
+
+Application scoping in SharePoint Embedded Copilot (SPE Copilot) involves defining the boundaries and context within which the tool operates, ensuring its features and capabilities are tailored to meet the specific needs of different applications. This process helps customize the Copilot's functionality, making it more effective and relevant for various use cases.
+
+When SPE Copilot users query the LLM, it will only have access to files that the `User+Application` have access to, which would mean a minima of a superset. Typically this will be the application that has been granted access over the container type which will typically be more data then just what the user has access to.
+
+![Venn Diagarm with 3P App access on left, SPE copilot in middle and consuming tenant user on right, overlapped area is what copilot can access](../images/speco-appscopingvenn.png)
+
+#### DiscoverabilityDisabled
+
+This flag prevents Copilot from discovering files in the specified container type. If you have an existing container type and are setting this value to false, please wait 24 hours to ensure the container type configuration is fully propagated before creating a new container, uploading files there, and trying out Copilot on folders/files of that new container.
+
+Here is an [example of setting the flag to false](https://learn.microsoft.com/en-us/powershell/module/sharepoint-online/set-spocontainertypeconfiguration?view=sharepoint-ps#examples)
+```powershell
+Set-SPOContainerTypeConfiguration -ContainerTypeId 4f0af585-8dcc-0000-223d-661eb2c604e4 -DiscoverabilityDisabled $false
+
+```
+
+Discoverability can also be disabled using the Visual Studio Code Sharepoint Embedded extension
+
+![Using the VS Code extension for SPE to set DiscoverabilityDisabled to false](../images/speco-vscodeextensiondisablediscovery.png)
+
+#### DataSources
+
+SPE Copilot has the ability to restrict the datasources it has access to, below are provided types and this [example](url-add) show's how to configure the SDK
+```typescript
+export type IDataSourcesProps =
+  | IFileDataSource
+  | IFolderDataSource
+  | IDocumentLibraryDataSource
+  | ISiteDataSource
+  | IWorkingSetDataSource
+  | IMeetingDataSource;
+
+export enum DataSourceType {
+  File = 'File',
+  Folder = 'Folder',
+  DocumentLibrary = 'DocumentLibrary',
+  Site = 'Site',
+  WorkingSet = 'WorkingSet',
+  Meeting = 'Meeting'
+}
+```
+### Semantic Index
+
+[Learn more about semantic index for M365 copilot here](https://learn.microsoft.com/en-us/microsoftsearch/semantic-index-for-copilot)
+
+The semantic index allows for quick and accurate searches based on the similarity of data. This means it can find the most relevant information not just by exact matches, but also by understanding the context and meaning.
+
+### RAG ( Retrevial-Augmented-Generation )

```

Comment:
Same as previous section, this is explaining a concept and nothing to do with usage, so move earlier